lib/mayaUsd/resources/scripts/mayaUsdAddMayaReference.mel

```diff
@@ -854,7 +809,7 @@ global proc addMayaReferenceToUsdUiCb(string $parent, string $selectedFile)
 
     int $radioExist = `radioCollection -exists namespaceRadioCollection`;
     int $useNamePrefix  = 1;
-    if($useNS)
+    if (true) // Use namespaces
```

Comment:
Can we just get rid of this?  Feels awkward to leave this dead code in.  Same at line 833.

Reply:
In this case (and the one below) I decided to leave the if here because of the variable and comment regarding namespaces. And in case we ever tried to compare this to the code from fileOptions.mel.